app/models/comment.rb

```diff
@@ -1,4 +1,15 @@
+require 'csv'
+
 class Comment < ApplicationRecord
   include Visible
   belongs_to :post
+
+  def self.to_csv
+    CSV.generate do |csv|
+      csv << ["Post title", "Comment author", "Comment body"]
```

Comment:
Just a nitpick, but the case gave the header in lowercase. Relevant if it's supposed to be machine-processed at some point.